tensorflow/java/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow;
+
+import java.util.Set;
+
+/**
+ * SavedModel representation once the SavedModel is loaded from storage.
+ */
+public class SavedModelBundle implements AutoCloseable {
+
+    SavedModelBundle(Graph graph, Session session, byte[] metaGraphDef) {
+        this.graph = graph;
+        this.session = session;
+        this.metaGraphDef = metaGraphDef;
+    }
+
+    /**
+     * Get the metagraphdef associated with the saved model bundle and tags used to load it.
+     */
+    public byte[] metaGraphDef() {
+        return metaGraphDef;
+    }
+
+    /**
+     * Get the graph loaded from the saved model bundle.
+     */
+    public Graph graph() {
+        return graph;
+    }
+
+    /**
+     * Get the session associated with the saved model bundle.
```

Comment:
Perhaps mention that the session has been initialized with any saved tensor values.
Something like:

Returns the Session with state (e.g., values of tensors) initialized from state saved in the directory provided to {@code load}.